src/login/logind-user.c

```diff
@@ -538,6 +539,23 @@ int user_check_linger_file(User *u) {
         return true;
 }
 
+static bool user_unit_active(User *u) {
+        const char *i;
+        int r;
+
```

Comment:
Maybe add
```C
assert(u->service);
assert(u->runtime_dir_service);
assert(u->slice);
```
here, because otherwise the loop will not do the expected thing.

Reply:
The requested change is implemented, and this comment got reattached in a random place. I'll mark this as resolved.